docs/index.md

```diff
@@ -0,0 +1,49 @@
+# MOTIS utl module documentation
```

Comment:
Maybe mention in that those logging function are an exception to the rule that we use [Aggregate Initialization](https://en.cppreference.com/w/cpp/language/aggregate_initialization) wherever possible. In principle you could write `utl::info{...}` which is not what we want.

Reply:
I added a note:
![image](https://github.com/user-attachments/assets/bd214902-cbdd-4837-86ac-2291edd36fd6)